test/Microsoft.ML.Tests/OnnxConversionTest.cs

```diff
@@ -1282,6 +1284,16 @@ public void WordTokenizerOnnxConversionTest()
                     var onnxResult = onnxTransformer.Transform(dataView);
                     var columnName = i == pipelines.Length - 1 ? "Tokens" : "NGrams";
                     CompareSelectedColumns<float>(columnName, columnName, transformedData, onnxResult, 3);
+
+                    VBuffer<ReadOnlyMemory<char>> mlNetSlots = default;
+                    VBuffer<ReadOnlyMemory<char>> onnxSlots= default;
+                    transformedData.Schema[columnName].GetSlotNames(ref mlNetSlots);
+                    onnxResult.Schema[columnName].GetSlotNames(ref onnxSlots);
+                    Assert.Equal(mlNetSlots.Length, onnxSlots.Length);
+                    var mlNetSlotNames = mlNetSlots.DenseValues().ToList();
+                    var onnxSlotNames = onnxSlots.DenseValues().ToList();
+                    for (int j = 0; j < mlNetSlots.Length; j++)
+                        Assert.Equal(mlNetSlotNames[j].ToString(), onnxSlotNames[j].ToString());
```

Comment:
>Equal [](start = 31, length = 5)

nit: I think Assert.Equal also has an overload for IEnumerables.

Reply:
I tried that already. But the Assert was firing even when all the strings were equal. Not sure why. 

---
In reply to: [381859560](https://github.com/dotnet/machinelearning/pull/4857#discussion_r381859560) [](ancestors = 381859560)